less/common/scaffolding.less

```diff
@@ -159,3 +159,13 @@ blockquote ol:last-child {
   font-size: 18px;
   color: @muted-more-color;
 }
+
+.visually-hidden {
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+}
```

Comment:
Can we make the checkbox the same dimensions as the switch itself ? we could make them parameters and then pass those values in (we'd have to relatively position the checkbox's parent and add `top:0;left:0;margin-top:-5px` as well to the checkbox)

Something I read here: https://www.sarasoueidan.com/blog/inclusively-hiding-and-styling-checkboxes-and-radio-buttons/#hiding-the-checkboxes-inclusively

Reply:
Good catch. I didn't realise that was how it worked!